test/test-project/src/test/bottomBar/webviewView-test.ts

```diff
@@ -0,0 +1,35 @@
+import { expect } from 'chai';
+import { BottomBarPanel, By, InputBox, WebviewView, Workbench } from 'vscode-extension-tester';
+
+describe('WebviewView', function () {
+
+    let webviewView: InstanceType<typeof WebviewView>;
+
+    before(async function () {
+        const prompt = await new Workbench().openCommandPrompt() as InputBox;
```

Comment:
whole prompt usage can be replaced with one line `await new Workbench().executeCommand('My Panel: Focus on My Panel View View');`